subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1202,6 +1255,22 @@ void ull_conn_tx_ack(struct ll_conn *conn, memq_link_t *link,
 	ll_tx_ack_put(conn->lll.handle, tx);
 }
 
+u8_t ull_conn_llcp_req(void *conn)
+{
+	struct ll_conn * const conn_hdr = conn;
+	if (conn_hdr->llcp_req != conn_hdr->llcp_ack) {
+		return BT_HCI_ERR_CMD_DISALLOWED;
```

Comment:
This makes good sense. req != ack, so we have an ongoing/pending operation, thus we shall deny this new request.